src/registrar/admin.py

```diff
@@ -2864,23 +2864,196 @@ def save_model(self, request, obj, form, change):
 
 
 class PortfolioAdmin(ListHeaderAdmin):
-
     change_form_template = "django/admin/portfolio_change_form.html"
+    fieldsets = [
+        # created_on is the created_at field, and portfolio_type is f"{organization_type} - {federal_type}"
+        (None, {"fields": ["portfolio_type", "organization_name", "creator", "created_on", "notes"]}),
+        # TODO - uncomment in #2521
+        # ("Portfolio members", {
+        #     "classes": ("collapse", "closed"),
+        #     "fields": ["administrators", "members"]}
+        # ),
+        ("Portfolio domains", {"classes": ("collapse", "closed"), "fields": ["domains", "domain_requests"]}),
+        ("Type of organization", {"fields": ["organization_type", "federal_type"]}),
+        (
+            "Organization name and mailing address",
+            {
+                "fields": [
+                    "federal_agency",
+                    "state_territory",
+                    "address_line1",
+                    "address_line2",
+                    "city",
+                    "zipcode",
+                    "urbanization",
+                ]
+            },
+        ),
+        ("Suborganizations", {"fields": ["suborganizations"]}),
+        ("Senior official", {"fields": ["senior_official"]}),
```

Comment:
SO is blanking out after page load (but the details are still there). Did not trace it.